update-ngxblocker

```diff
@@ -272,16 +281,24 @@ send_email() {
 		print_message "${BOLDYELLOW}WARN${RESET}: missing mail command => ${BOLDWHITE}disabling emails${RESET}.\n\n"
 	fi
 }
+send_email_via_mailgun() {
```

Comment:
please insert a blank line above:

 `send_email_via_mailgun() {` 

to give spacing after the function above it

Reply:
done